### PR TITLE
Implement split images

### PR DESCRIPTION
**Pull request by stgraber**

### CODE REVIEW

**Comment by stgraber** [comment]
Per the spec, the rootfs tarball should be just the rootfs without any prefix.

**Comment by stgraber** [comment]
Where did you see snake case variables in my Go code? I just re-read through the diff and couldn't spot any newly introduced ones.

There are a bunch in lxd-images but that file is python3 where snake case is the recommendation and camel case is the exception.

**Comment by pcdummy** [comment]
I'm sorry mate, yes it's in scripts/lxd-images.

File: lxc/config.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ func (c *configCmd) usage() string {
 			"lxc config unset key                                   Unset server configuration key\n" +
 			"lxc config show [remote:]<container>                   Show container configuration\n" +
 			"lxc config trust list [remote]                         List all trusted certs.\n" +
-			"lxc config trust add [remote] [certfile.crt]           Add certfile.crt to trusted hosts.\n" +
+			"lxc config trust add [remote] <certfile.crt>           Add certfile.crt to trusted hosts.\n" +
```
Comment (tych0) [inline lxc/config.go]: What's isn't the cert optional here? Why the change?

Comment (stgraber) [inline lxc/config.go]: Just updating to make it match the spec.

What would "lxc config trust add" without any argument mean? It can't possibly at a certificate to the trust store without being provided a certificate.
